docs/companion.md

```diff
@@ -463,6 +463,10 @@ which has only the secret, nothing else.
 
 :::
 
+##### `s3.endpoint` `COMPANION_AWS_ENDPOINT`
+
+URL to the S3 (compatible) service.
```

Comment:
```suggestion
URL to a custom S3 (compatible) service.
```

Reply:
custom implies that it's optional? is that the case? Does it have a default?

Reply:
I believe the default lies somewhere inside aws sdk. And it’s probably amazon-aws-something.com